src/pages/dmt5/_margin_calculator.js

```diff
@@ -0,0 +1,292 @@
+import React, { useState, useEffect } from 'react'
+import Proptypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import styled from 'styled-components'
+import device, { size } from 'themes/device'
+import { Box, Flex, SectionContainer } from 'components/containers'
+import { Carousel, Header, LinkText, QueryImage, Text } from 'components/elements'
+import { LinkButton } from 'components/form'
+import { Localize } from 'components/localization'
+import { isBrowser } from 'common/utility'
+
+const query = graphql`
+    query {
+        margin_calculator: file(relativePath: { eq: "dmt5-margin-calculator.png" }) {
+            ...fadeIn
+        }
+        margin_calculator_mobile: file(relativePath: { eq: "dmt5-margin-calculator-mobile.png" }) {
+            ...fadeIn
+        }
+        swap_calculator: file(relativePath: { eq: "dmt5-swap-calculator.png" }) {
+            ...fadeIn
+        }
+        swap_calculator_mobile: file(relativePath: { eq: "dmt5-swap-calculator-mobile.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const ImageWrapper = styled.div`
+    max-width: 39.2rem;
+    width: 100%;
+    height: 38.6rem;
+    object-fit: contain;
+    margin-bottom: 2.4rem;
+
+    @media ${device.mobileL} {
+        max-width: 232px;
+        width: 100%;
+        margin: 0 0 24px;
+        height: unset;
+
+        div {
+            max-width: 232px;
+            width: 100%;
+        }
+    }
+`
+
+const MainHeader = styled(Header)`
+    font-size: 48px;
+    margin: 0 0 12px;
+    line-height: 1.25;
+    text-align: left;
+    color: var(--color-black-3);
+    @media ${device.mobileL} {
+        font-size: 32px;
+        margin-bottom: 24px;
+        padding: 0 16px;
+        text-align: center;
+    }
+`
+
+const SubHeader = styled(Header)`
+    font-size: 24px;
+    color: var(--color-black-3);
+    @media ${device.mobileL} {
+        font-size: 20px;
+        margin-bottom: 8px;
+    }
+`
+
+const StyledBox = styled(Box)`
+    max-width: 508px;
+    margin: 11.9rem 4rem 0 16rem;
+    @media ${device.mobileL} {
+        max-width: 100%;
+        margin: 4rem 1.6rem 2.4rem 1.6rem;
+    }
+`
+
+const StyledText = styled(Text)`
+    text-align: ${(props) => (props.align ? 'center' : 'left')};
+    margin-bottom: 24px;
+    @media ${device.mobileL} {
+        font-size: 16px;
+    }
+`
+
+const CardText = styled(StyledText)`
+    margin: 0.8rem 4rem 2.4rem 4rem;
+    @media ${device.mobileL} {
+        margin: 0 0 24px 0;
+    }
+`
+
+const StyledLinkButton = styled(LinkButton)`
+    padding: 1rem 1.6rem;
+    font-size: 14px;
+    height: 4rem;
+    width: 21rem;
```

Comment:
I think we should set this to `auto` to cater for all translations. Could set a `min-width` of 21rem if required.